File: Loop/Prime_Number_Check.py
number=int(input("Enter the number: "))
isPrime=True
for num  in range (2, number):
    if number%num==0:
        print("The nummber is divisible by the number: ", num)
        isPrime=False
        break
    
if isPrime==False:
    print("The number is not Prime!!")
else:
    print("The number is prime")
    
 
        

File: Loop/primenumbercheck.cpp
//Prime number checker

#include<iostream>
using namespace std;
int main()
{
    int number,i=2;
    cout<<"Enter a number:";
    cin>>number;

    while(i<number)
    {
        if(number%i==0)
        {
       cout<<"The number is not a prime number "<<endl;
       break;
        }
        else
        {
            cout<<"The number is a prime number"<<endl;
            break;
        }
        
        return 0;
}
}